.drone.star

```diff
@@ -366,6 +492,260 @@ def phan():
+				'kind': 'pipeline',
+				'type': 'docker',
+				'name': name,
+				'workspace' : {
+					'base': '/drone',
+					'path': 'src'
+				},
+				'steps':
+					cacheRestore() +
+					composerInstall(phpVersion) +
+					yarnInstall(phpVersion) +
+					installServer(phpVersion, db, params['logLevel'], params['useHttps']) +
+					setupLocalStorage(phpVersion) +
+					fixPermissions(phpVersion, False) +
+					createShare(phpVersion) +
+					owncloudLog('server', 'src') +
+					[
+						{
+							'name': 'old-endpoint',
+							'image': image,
+							'pull': 'always',
+							'environment': environment,
+							'commands': [
+								'source .env',
+								'export LITMUS_URL="https://server/remote.php/webdav"',
+								litmusCommand,
+							]
+						},
+						{
+							'name': 'new-endpoint',
+							'image': image,
+							'pull': 'always',
+							'environment': environment,
+							'commands': [
+								'source .env',
+								'export LITMUS_URL="https://server/remote.php/dav/files/admin"',
+								litmusCommand,
+							]
+						},
+						{
+							'name': 'new-mount',
+							'image': image,
+							'pull': 'always',
+							'environment': environment,
+							'commands': [
+								'source .env',
+								'export LITMUS_URL="https://server/remote.php/dav/files/admin/local_storage/"',
+								litmusCommand,
+							]
+						},
+						{
+							'name': 'old-mount',
+							'image': image,
+							'pull': 'always',
+							'environment': environment,
+							'commands': [
+								'source .env',
+								'export LITMUS_URL="https://server/remote.php/webdav/local_storage/"',
+								litmusCommand,
+							]
+						},
+						{
+							'name': 'new-shared',
+							'image': image,
+							'pull': 'always',
+							'environment': environment,
+							'commands': [
+								'source .env',
+								'export LITMUS_URL="https://server/remote.php/dav/files/admin/new_folder/"',
+								litmusCommand,
+							]
+						},
+						{
+							'name': 'old-shared',
+							'image': image,
+							'pull': 'always',
+							'environment': environment,
+							'commands': [
+								'source .env',
+								'export LITMUS_URL="https://server/remote.php/webdav/new_folder/"',
+								litmusCommand,
+							]
+						},
+						{
+							'name': 'public-share',
+							'image': image,
+							'pull': 'always',
+							'environment': {
+								'LITMUS_PASSWORD': 'admin',
+								'LITMUS_USERNAME': 'admin',
+								'TESTS': 'basic copymove http',
+							},
+							'commands': [
+								'source .env',
+								'export LITMUS_URL=\'https://server/remote.php/dav/public-files/\'$PUBLIC_TOKEN',
+								litmusCommand,
+							]
+						},
+					],
+				'services':
+					databaseService(db) +
+					owncloudService(phpVersion, 'server', '/drone/src', params['useHttps']),
+				'depends_on': [],
+				'trigger': {
+					'ref': [
+						'refs/pull/**',
+						'refs/tags/**'
+					]
+				}
+			}
+
+			pipelines.append(result)
+
+	return pipelines
+
+def dav():
```

Comment:
probably this is not called and can be deleted

Reply:
This one also has been called in `stagePipelines`